Recurrent issue with test failure

The test is one which gets the value of the Encrypted property. It doesn't fail very often, but it does occasionally, with this trace:

```
proxy_object = <ProxyObject wrapping <dbus._dbus.SystemBus (system) at 0x7fb718df6950> :1.7374 / at 0x7fb718dabe90>

    def dbus_func(proxy_object):  # pragma: no cover
        """
        The property getter.
    
        :raises DPClientInvocationError:
        """
        try:
            return proxy_object.Get(
                interface_name,
                name,
                dbus_interface=dbus.PROPERTIES_IFACE,
>               timeout=timeout)

../../stratis-deps/dbus-python-client-gen/src/dbus_python_client_gen/_invokers.py:68: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <dbus.proxies._ProxyMethod object at 0x7fb718dab510>
args = ('org.storage.stratis2.pool.r1', 'Encrypted')
keywords = {'timeout': 120}, reply_handler = None, error_handler = None
ignore_reply = False, signature = None
dbus_interface = 'org.freedesktop.DBus.Properties'
key = 'org.freedesktop.DBus.Properties.Get'

    def __call__(self, *args, **keywords):
        reply_handler = keywords.pop('reply_handler', None)
        error_handler = keywords.pop('error_handler', None)
        ignore_reply = keywords.pop('ignore_reply', False)
        signature = keywords.pop('signature', None)
    
        if reply_handler is not None or error_handler is not None:
            if reply_handler is None:
                raise MissingReplyHandlerException()
            elif error_handler is None:
                raise MissingErrorHandlerException()
            elif ignore_reply:
                raise TypeError('ignore_reply and reply_handler cannot be '
                                'used together')
    
        dbus_interface = keywords.pop('dbus_interface', self._dbus_interface)
    
        if signature is None:
            if dbus_interface is None:
                key = self._method_name
            else:
                key = dbus_interface + '.' + self._method_name
    
            signature = self._proxy._introspect_method_map.get(key, None)
    
        if ignore_reply or reply_handler is not None:
            self._connection.call_async(self._named_service,
                                        self._object_path,
                                        dbus_interface,
                                        self._method_name,
                                        signature,
                                        args,
                                        reply_handler,
                                        error_handler,
                                        **keywords)
        else:
            return self._connection.call_blocking(self._named_service,
                                                  self._object_path,
                                                  dbus_interface,
                                                  self._method_name,
                                                  signature,
                                                  args,
>                                                 **keywords)

/usr/lib64/python3.7/site-packages/dbus/proxies.py:145: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <dbus._dbus.SystemBus (system) at 0x7fb718df6950>
bus_name = dbus.String(':1.7374'), object_path = dbus.ObjectPath('/')
dbus_interface = 'org.freedesktop.DBus.Properties', method = 'Get'
signature = None, args = ('org.storage.stratis2.pool.r1', 'Encrypted')
timeout = 120, byte_arrays = False, kwargs = {}
get_args_opts = {'byte_arrays': False}
message = <dbus.lowlevel.MethodCallMessage path: /, iface: org.freedesktop.DBus.Properties, member: Get dest: :1.7374>

    def call_blocking(self, bus_name, object_path, dbus_interface, method,
                      signature, args, timeout=-1.0,
                      byte_arrays=False, **kwargs):
        """Call the given method, synchronously.
        :Since: 0.81.0
        """
        if object_path == LOCAL_PATH:
            raise DBusException('Methods may not be called on the reserved '
                                'path %s' % LOCAL_PATH)
        if dbus_interface == LOCAL_IFACE:
            raise DBusException('Methods may not be called on the reserved '
                                'interface %s' % LOCAL_IFACE)
        # no need to validate other args - MethodCallMessage ctor will do
    
        get_args_opts = dict(byte_arrays=byte_arrays)
        if is_py2:
            get_args_opts['utf8_strings'] = kwargs.get('utf8_strings', False)
        elif 'utf8_strings' in kwargs:
            raise TypeError("unexpected keyword argument 'utf8_strings'")
    
        message = MethodCallMessage(destination=bus_name,
                                    path=object_path,
                                    interface=dbus_interface,
                                    method=method)
        # Add the arguments to the function
        try:
            message.append(signature=signature, *args)
        except Exception as e:
            logging.basicConfig()
            _logger.error('Unable to set arguments %r according to '
                          'signature %r: %s: %s',
                          args, signature, e.__class__, e)
            raise
    
        # make a blocking call
        reply_message = self.send_message_with_reply_and_block(
>           message, timeout)
E       dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist

/usr/lib64/python3.7/site-packages/dbus/connection.py:651: DBusException

The above exception was the direct cause of the following exception:

self = <tests.dbus.pool.test_properties.SetNameTestCase testMethod=testProps>

    def testProps(self):
        """
        Test reading some pool properties.
        """
        pool = get_object(self._pool_object_path)
>       is_encrypted = PoolR1.Properties.Encrypted.Get(pool)

tests/dbus/pool/test_properties.py:54: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

proxy_object = <ProxyObject wrapping <dbus._dbus.SystemBus (system) at 0x7fb718df6950> :1.7374 / at 0x7fb718dabe90>

    def dbus_func(proxy_object):  # pragma: no cover
        """
        The property getter.
    
        :raises DPClientInvocationError:
        """
        try:
            return proxy_object.Get(
                interface_name,
                name,
                dbus_interface=dbus.PROPERTIES_IFACE,
                timeout=timeout)
        except dbus.DBusException as err:
            err_msg = ("Error while getting value for property \"%s\" "
                       "belonging to interface \"%s\"") % (name,
                                                           interface_name)
            raise DPClientInvocationError(
                err_msg, interface_name,
>               DPClientGetPropertyContext(name)) from err
E           dbus_python_client_gen._errors.DPClientInvocationError: Error while getting value for property "Encrypted" belonging to interface "org.storage.stratis2.pool.r1"

../../stratis-deps/dbus-python-client-gen/src/dbus_python_client_gen/_invokers.py:75: DPClientInvocationError
```

--- a/tests/client-dbus/tests/dbus/pool/test_properties.py
+++ b/tests/client-dbus/tests/dbus/pool/test_properties.py
@@ -21,12 +21,12 @@ from stratisd_client_dbus._constants import TOP_OBJECT
 
 from .._misc import SimTestCase, device_name_list
 
-_DEVICE_STRATEGY = device_name_list()
+_DEVICE_STRATEGY = device_name_list(1)
 
 
 class PropertyTestCase(SimTestCase):
     """
-    Set up a pool with a name and one filesystem.
+    Set up a pool with at least one device.
     """
 
     _POOLNAME = "deadpool"

--- a/tests/client-dbus/tests/dbus/pool/test_properties.py
+++ b/tests/client-dbus/tests/dbus/pool/test_properties.py
@@ -24,7 +24,7 @@ from .._misc import SimTestCase, device_name_list
 _DEVICE_STRATEGY = device_name_list()
 
 
-class SetNameTestCase(SimTestCase):
+class PropertyTestCase(SimTestCase):
     """
     Set up a pool with a name and one filesystem.
     """

